nucliadb/nucliadb/search/search/utils.py

```diff
@@ -63,7 +69,33 @@ def parse_sort_options(item: SearchRequest) -> SortOptions:
     return sort_options
 
 
-def is_empty_query(request: SearchRequest) -> bool:
+def is_empty_query(request: BaseSearchRequest) -> bool:
     return len(request.query) == 0 and (
         request.advanced_query is None or len(request.advanced_query) == 0
     )
+
+
+def has_user_vectors(request: BaseSearchRequest) -> bool:
+    return request.vector is not None and len(request.vector) > 0
+
+
+def is_exact_match_only_query(request: BaseSearchRequest) -> bool:
+    """
+    '"something"' -> True
+    'foo "something" else' -> False
+    """
+    query_split = request.query.strip().split('"')
```

Comment:
Would a better check just be something like:

```
query = request.query.strip()
return len(query) > 0 and query[0] == '"' and query[-1] == '"'
```

?

Reply:
changed!